<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserReportIssue</class>
 <widget class="QDialog" name="UserReportIssue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(104, 161, 229);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>186</y>
     <width>591</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
     border-radius: 5px; 
	background-color: rgb(242, 248, 253);
 } </string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253, 0)</string>
      </property>
      <property name="text">
       <string>Please fill out all forms</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
        </property>
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditFirstName">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
        </property>
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLastName">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditEmail">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
        </property>
        <property name="text">
         <string>Write message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditMessage">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(104, 161, 229);</string>
   </property>
   <property name="text">
    <string> Report Issue</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
     border-radius: 5px; 
	background-color: rgb(242, 248, 253);
 } </string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>9</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 253,0)</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>10</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 253, 0)</string>
    </property>
    <property name="text">
     <string> Report Issue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHome">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0,0.1); 
	border: none; 
	color: rgb(0, 0, 0); 
	font: 14px;
} 

QPushButton:hover { 
	background-color: rgb(0, 0, 0, 0.1); 
	border: none; 
	color: rgb(0, 43, 255); 
}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>710</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
    padding: 5 20px;
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRequest">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>710</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
    padding: 5 20px;
}
</string>
   </property>
   <property name="text">
    <string>Request</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>75</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(104, 161, 229);</string>
      </property>
      <property name="text">
       <string>My Health Account </string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>368</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border-radius: 3px; 
	background-color: rgb(242, 248, 253);
 } </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelIconUser">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(242, 248, 253,0)</string>
         </property>
         <property name="text">
          <string>Icon</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="labelUserName">
           <property name="font">
            <font>
             <family>.AppleSystemUIFont</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
           </property>
           <property name="text">
            <string>User Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonViewProfile">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0,0.1); 
	border: none; 
	color: rgb(0, 0, 0); 
	font: 12px;
} 

QPushButton:hover { 
	background-color: rgb(0, 0, 0, 0.1); 
	border: none; 
	color: rgb(0, 43, 255); 
}</string>
           </property>
           <property name="text">
            <string>View Profile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButtonLogout">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
	padding: 5 20px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
        </property>
        <property name="text">
         <string>Log out</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
